### DISCUSSION TITLE
Custom Report Not Generating With Department Info

### DISCUSSION THREAD

**Discussion by TraumaSK** [snipe/snipe-it#12229] [General]
Hello friends!  

Just a note to say that it looks like custom report generation doesn't include any department info about an asset, even if there is a department associated with the user the item is assigned to.  Unsure if perhaps I'm simply doing something wrong, or if this is just a small bug etc.  Wanted to report it here.

1 - Created a few test users and test departments to associate them with.
2 - Created and assigned an asset to the users.
3 - Ran custom report, including the department as a field I wanted in the report.

![image](https://user-images.githubusercontent.com/28709564/206497029-6b840a06-9fd4-4cf0-8ec4-e837af924ac6.png)
![image](https://user-images.githubusercontent.com/28709564/206497323-7703ddfe-548a-4cea-87ca-0272485d149e.png)
![image](https://user-images.githubusercontent.com/28709564/206497525-dba5305c-21ba-4b82-bb66-0a0986852243.png)

**Comment by kjshearer** [comment]
Same issue here. I reported several weeks ago and they confirmed it was a bug. Waiting for them to implement the fix.
